### PR TITLE
Changelog for 0.7.0

### PR DESCRIPTION
**Pull request by j-stephan**
This PR updates the CHANGELOG.md file for the next release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,61 @@ All notable changes to this project will be documented in this file.
...
+- the CMake option `ALPAKA_CUDA_NVCC_EXPT_EXTENDED_LAMBDA` has been replaced by `ALPAKA_CUDA_EXPT_EXTENDED_LAMBDA` #1289
+
+### Test cases / CI:
+- enabled OpenMP back-ends for more Visual Studio builds #1219
+- fixed gh-pages #1230
+- added ICPC / ICC 2021.x to CI #1235
+- now using Ubuntu 18.04 for GCC 5 and GCC 6 builds #1252
+- fixed queue test case which incorrectly assumed that the checked queue will always be empty #1266
+- fixed deadlock in Ubuntu 20.04 container #1270
+- added support for CMake 3.20 #1283
```
Comment (sbastrakov) [inline CHANGELOG.md]: Should this one be moved to the compatibility section?

Comment (j-stephan) [reply CHANGELOG.md]: This just means that we test CMake 3.20 in our CI. There is no special CMake 3.20 compatibility change in our build system that I'm aware of. I can change this sentence to make this clearer.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,61 @@ All notable changes to this project will be documented in this file.
...
+- clang 11 is now supported #1310
+
+### Bug Fixes:
+- fixed static shared memory allocation for OpenMP 5 back-end #1254
+- fixed static shared memory allocation for CPU back-ends #1258
+- fixed `ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED` being checked without being defined #1259
+- fixed `BlockSharedMemStMemberImpl::getVarPtr` for last variable #1280
+- fixed static shared memory alignment #1282
+- fixed `BlockSharedMemStMemberImpl::getVarPtr` for last variable #1280
+- fixed static shared memory alignment #1282
```
Comment (bernhardmgruber) [inline CHANGELOG.md]: These look like duplicates to me.

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,61 @@ All notable changes to this project will be documented in this file.
...
+- fixed inverted assertion in `DeclareSharedVar` #1303
+- fixed OpenMP scheduler support #1307 #1309 #1316
+
+### New Features:
+- math functions are now ADL friendly #1248
```
Comment (bernhardmgruber) [inline CHANGELOG.md]:
```suggestion
- alpaka math functions, when no specialization is provided, will now fall back to ADL to find a candidate #1248
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,61 @@ All notable changes to this project will be documented in this file.
...
+- added `Front` and `Contains` type list meta functions #1306
+
+### Misc:
+- alpaka's CMake build system now uses CMake's first-class CUDA support #1146
+- updated documentation for clang-format usage #1222
+- increased the static shared memory size to 47 KiB #1247
+- fixed table markup in README.md #1256
+- return values of `empty(queue)` and `isComplete(event)` are now correctly documented #1265
+- added example showcasing how to specialize kernels for particular back-ends #1271
+- removed section comments #1275
```
Comment (bernhardmgruber) [inline CHANGELOG.md:37]: I think I would not even list this, because it is a non-functional change.

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,61 @@ All notable changes to this project will be documented in this file.
...
+- fixed static shared memory allocation for OpenMP 5 back-end #1254
+- fixed static shared memory allocation for CPU back-ends #1258
+- fixed `ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED` being checked without being defined #1259
+- fixed `BlockSharedMemStMemberImpl::getVarPtr` for last variable #1280
+- fixed static shared memory alignment #1282
+- fixed `BlockSharedMemStMemberImpl::getVarPtr` for last variable #1280
+- fixed static shared memory alignment #1282
+- fixed inverted assertion in `DeclareSharedVar` #1303
+- fixed OpenMP scheduler support #1307 #1309 #1316
+
```
Comment (bernhardmgruber) [inline CHANGELOG.md:22]: The PR #1129 seems to be missing. Please add something like:
`forward ALPAKA_CXX_STANDARD to nvcc on Windows: #1129`

**Review by bernhardmgruber — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 5 above)
Comment (psychocoderHPC) [reply CHANGELOG.md:22]: Issues will not be part of the CHANGELOG, only PRs which solves issues. The PR to solve #1129 should be #1130

Comment (bernhardmgruber) [reply CHANGELOG.md:22]: Right! So it should be #1130. Thx!

Comment (j-stephan) [reply CHANGELOG.md:22]: #1130 is already part of 0.6.0 (according to the assigned milestone). Did this change and it is actually part of 0.7.0?

Comment (bernhardmgruber) [reply CHANGELOG.md:22]: Then the issue #1129 was wrongly assigned to 0.7.0, I changed it to 0.6.0.

**Review by bernhardmgruber — APPROVED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,58 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [0.7.0] - 2021-07-01
```
Comment (psychocoderHPC) [inline CHANGELOG.md:6]: This is a good day for a release :heart: . This is my 10th jubilee at HZDR^^

File: CHANGELOG.md [hunk 7]
```diff
@@ -3,6 +3,58 @@ All notable changes to this project will be documented in this file.
...
+- clang 11 is now supported #1310
+
+### Bug Fixes:
+- fixed static shared memory allocation for OpenMP 5 back-end #1254
+- fixed static shared memory allocation for CPU back-ends #1258
+- fixed `ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED` being checked without being defined #1259
+- fixed `BlockSharedMemStMemberImpl::getVarPtr` for last variable #1280
+- fixed static shared memory alignment #1282
+- fixed inverted assertion in `DeclareSharedVar` #1303
+- fixed OpenMP scheduler support #1307 #1309 #1316
```
Comment (psychocoderHPC) [inline CHANGELOG.md:21]: This is a bug fix and also a breaking change, we should mention it in the `Breaking Changes` too.

**Review by psychocoderHPC — APPROVED** [review]
I have only one optional suggestion

File: CHANGELOG.md [hunk 8]
```diff
@@ -3,6 +3,59 @@ All notable changes to this project will be documented in this file.
...
+- updated cheatsheet (added warp info, fixed names) #1281
+
+### Breaking Changes:
+- alpaka now requires CMake 3.18 or newer #1146
+- the CUDA and HIP back-ends no longer enable fast-math by default #1285
+- the CMake options `ALPAKA_CUDA_FAST_MATH` and `ALPAKA_HIP_FAST_MATH` have been replaced by `ALPAKA_FAST_MATH` #1289
+- the CMake options `ALPAKA_CUDA_FTZ` and `ALPAKA_HIP_FTZ` have been replaced by `ALPAKA_FTZ` #1289
+- the CMake option `ALPAKA_CUDA_NVCC_SEPARABLE_COMPILATION` has been replaced by the native CMake property `CUDA_SEPARABLE_COMPILATION` #1289
+- the CMake option `ALPAKA_CUDA_NVCC_EXPT_EXTENDED_LAMBDA` has been replaced by `ALPAKA_CUDA_EXPT_EXTENDED_LAMBDA` #1289
+- fixing the OpenMP scheduler behaviour changed the default behaviour #1309
```
Comment (j-stephan) [inline CHANGELOG.md:47]: @psychocoderHPC Is this additional line what you intended?

File: CHANGELOG.md (hunk 6 above)
Comment (SimeonEhrig) [reply CHANGELOG.md:6]: I think, this is a good day to finish the home office and come back to the HZDR. Cake time :grin:

**Pull request merged**